### DISCUSSION TITLE
debugView on Archive ios IPA

### DISCUSSION THREAD

**Discussion by oottoohh** [invertase/react-native-firebase#7508] [Q&A]
Hi everyone, is it possible to using debugView features with ipa installed on device ? i try many ways, still fail debugView Mode can not detect my device but running well when i try running directly to device using XCode. Thanks Advance

**Comment by mikehardy** [comment]
This question is not answerable as stated

https://stackoverflow.com/help/how-to-ask
https://stackoverflow.com/help/minimal-reproducible-example
